test/helpers/json-api.js

```diff
@@ -10,8 +10,8 @@ function getResource(data, type) {
   return {
     id: data.id,
     type,
-    attributes: _.omit(data, 'id'),
-    relationships: {},
+    attributes: _.extend(_.omit(data, 'id'), attributes),
+    relationships: relationships || {},
```

Comment:
There's a few test files that were adding `fx.included.relationship` data along with `fx.included.attributes`. So I added `relationships` as another function argument.

Reply:
This causes this test to fail: https://cloud.cypress.io/projects/ep9zr6/runs/5614/test-results/1cd5e357-fc6f-4a44-96f2-c17d0ffb2345

Reply:
I fixed that failing test. But I sense this is going to cause problems.